packages/embed/src/css/includes/_spinner.scss

```diff
@@ -11,7 +13,7 @@ $spinner-sides: 1 !default;
   $diameter: $spinner-diameter,
   $sides: $spinner-sides
 ) {
-  $spinner-margin: $diameter / 2px * -1px;
+  $spinner-margin: math.div($diameter, 2px) * -1px;
```

Comment:
nice!

Reply:
FYI it is a breaking change in latest sass. Unless you change this, it produces a warning:
https://sass-lang.com/documentation/breaking-changes/slash-div